bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/library/unit/SmartHomeUnits.java

```diff
@@ -164,6 +171,9 @@ public String getName() {
         SimpleUnitFormat.getInstance().label(MILLIBAR, "mbar");
         SimpleUnitFormat.getInstance().label(KNOT, KNOT.getSymbol());
         SimpleUnitFormat.getInstance().label(DOBSON_UNIT, "DU");
+        SimpleUnitFormat.getInstance().label(CUBICMETRE_PER_SECOND, "m³/s");
+        SimpleUnitFormat.getInstance().label(LITRE_PER_MINUTE, "L/min");
```

Comment:
The unit symbol for Units.LITRE is „l“, not „L“. We should use the same here.

Reply:
I understand your point, however `L` is also widely accepted and I am not sure if its not more popular in some circles due to readability. It is easier to read it in this for that l (`l`) which is quite similar to I (`I`). I can change to `l/min`, however at the same time I would ask why not going with `L` in place you mentioned?

Reply:
Wikipedia says that both are allowed. „l“ was the original symbol, in 1979 „L“
was allowed as alternative symbol. 

The upper-case L is mostly used in US/Canada while the lower-case l is used in Europe. I‘m not sure if we can allow both, what I would suggest.

Reply:
UoM currently does not support multiple formats for the same unit. I would prefer the `l` variant.